decidim-core/app/helpers/decidim/cells_helper.rb

```diff
@@ -35,6 +35,7 @@ def flaggable?
     end
 
     def user_flaggable?
+      return if (try(:profile_holder) || try(:profile_user) || try(:model))&.blocked?
```

Comment:
avoid using safe navigation ( `&,` ), and use the `.try` instead
In some certain cases, An author can be the Organization (ex: Official meetings)